<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsChoicesDialog</class>
 <widget class="QDialog" name="clsChoicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choices</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <spacer name="spcrTitle">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QLabel" name="lblQuestion">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;What would you like to do?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <widget class="QPushButton" name="btnCancel">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <layout class="QHBoxLayout" name="btnLeftRightHold">
     <item>
      <widget class="QPushButton" name="btnLeftHold">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spcrLeftRightHold">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRightHold">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="6" colspan="2">
    <layout class="QHBoxLayout" name="btnLeftRightRapidFire">
     <item>
      <widget class="QPushButton" name="btnRapidFireLeft">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spcrLeftRightRapidFire">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRapidFireRight">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QPushButton" name="btnBuildHorizontal">
     <property name="text">
      <string>Build horizontally</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QPushButton" name="btnNerdPole">
     <property name="text">
      <string>Nerdpole</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QPushButton" name="btnMine">
     <property name="text">
      <string>Mine</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QPushButton" name="btnFish">
     <property name="text">
      <string>Fish</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QLabel" name="lblRapidFire">
     <property name="text">
      <string>Rapid fire button:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QLabel" name="lblHoldBtn">
     <property name="text">
      <string>Hold button:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
